src/index.css

```
@@ -1,3 +1,7 @@
+@tailwind base;
```

Stylelint (17.9.0)

[error] Unexpected unknown at-rule "@tailwind" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)